apps/dashboard/src/components/selects/NetworkSelectorButton.tsx

```diff
@@ -196,7 +196,7 @@ export const NetworkSelectorButton: React.FC<NetworkSelectorButtonProps> = ({
           });
         }}
       >
-        <ChainIcon ipfsSrc={chain?.icon?.url} size={20} />
+        <ChainIcon ipfsSrc={chain?.icon?.url} className="size-5" />
```

Comment:
btw this is one of the things we should switch to using the react sdk components for!